File: Example 7.js
const foods = ['Sate', 'Bakso', 'Dimsum', 'Rames'];
const drinks = ['Jeruk','Teh', 'Kelapa','Cendol'];

function logPairs(arrayl, array2, words) {
    let counter = 0;
    for (let i = 0; i < arrayl.length; i++) { //0(n)
        for (let j = 0; j < array2.length; j++) { //0(n = m)
            counter++; //0(n = m)
            console.log(words + ' ' + counter + ' ' + arrayl[i] + ' dan ' + array2[j]); //0(n = m)
        }
    }
}

logPairs(foods, drinks, 'Menu');

File: Example 7.go
package main

import "fmt"

func main() {
    foods := []string{"Sate", "Bakso", "Dimsum", "Rames"}
    drinks := []string{"Jeruk", "Teh", "Kelapa", "Cendol"}
    counter := 0

    for i := 0; i < len(foods); i++ {
        for j := 0; j < len(drinks); j++ {
            counter++
            fmt.Printf("Menu %d: %s dan %s\n", counter, foods[i], drinks[j])
        }
    }
}